docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "System Table Configuration"
+date: 2024-01-23T16:30:21+05:30
+---
+
+## Configuration changes required for System Table Integration
+
+There is no unique configuration process for System Table. The only change required to the configuration for integration
+is below - 
+
+* **auditlogprefix_source_path** - Instead of adding a fully qualified path (s3 or GC) for auditlog, 
+add keyword **system** in this column. This will enable the system table integration.
+
+**For migrating to system table from legacy above same congifuration changes can be used.**
```

Comment:
We need to remove this line. Legacy is no longer supported

Reply:
done